sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/encoders/ExpressionEncoderSuite.scala

```diff
@@ -452,6 +452,10 @@ class ExpressionEncoderSuite extends CodegenInterpretedPlanTest with AnalysisTes
     }
   }
 
+  encodeDecodeTest("foo" -> 1L, "copied encoder") {
```

Comment:
@zsxwing, can we move this test up over `testOverflowingBigNumeric`? I think private helper method look intentionally grouped below here.